libr/anal/esil.c

```diff
@@ -3028,149 +3048,157 @@ R_API int r_anal_esil_condition(RAnalEsil *esil, const char *str) {
+	OP ("}{", esil_nop, 0, 0, OT_CTR);
+	OP ("|", esil_or, 1, 2, OT_MATH);
+	OP ("|=", esil_oreq, 0, 2, OT_MATH | OT_REGW);
+	OP ("!", esil_neg, 1, 1, OT_MATH);
+	OP ("!=", esil_negeq, 0, 1, OT_MATH | OT_REGW);
+	OP ("=", esil_eq, 0, 2, OT_REGW);
+	OP (":=", esil_weak_eq, 0, 2, OT_REGW);
+	OP ("*", esil_mul, 1, 2, OT_MATH);
+	OP ("*=", esil_muleq, 0, 2, OT_MATH | OT_REGW);
+	OP ("^", esil_xor, 1, 2, OT_MATH);
+	OP ("^=", esil_xoreq, 0, 2, OT_MATH | OT_REGW);
+	OP ("+", esil_add, 1, 2, OT_MATH);
+	OP ("+=", esil_addeq, 0, 2, OT_MATH | OT_REGW);
+	OP ("++", esil_inc, 1, 1, OT_MATH);
+	OP ("++=", esil_inceq, 0, 1, OT_MATH | OT_REGW);
+	OP ("-", esil_sub, 1, 2, OT_MATH);
+	OP ("-=", esil_subeq, 0, 2, OT_MATH | OT_REGW);
+	OP ("--", esil_dec, 1, 1, OT_MATH);
+	OP ("--=", esil_deceq, 0, 1, OT_MATH | OT_REGW);
+	OP ("/", esil_div, 1, 2, OT_MATH);
+	OP ("/=", esil_diveq, 0, 2, OT_MATH | OT_REGW);
+	OP ("%", esil_mod, 1, 2, OT_MATH);
+	OP ("%=", esil_modeq, 0, 2, OT_MATH | OT_REGW);
+	OP ("=[]", esil_poke, 0, 2, OT_MEMW);
+	OP ("=[1]", esil_poke1, 0, 2, OT_MEMW);
+	OP ("=[2]", esil_poke2, 0, 2, OT_MEMW);
+	OP ("=[3]", esil_poke3, 0, 2, OT_MEMW);
+	OP ("=[4]", esil_poke4, 0, 2, OT_MEMW);
+	OP ("=[8]", esil_poke8, 0, 2, OT_MEMW);
+	OP ("=[16]", esil_poke16, 0, 2, OT_MEMW);
+	OP ("|=[]", esil_mem_oreq, 0, 2, OT_MATH | OT_MEMR | OT_MEMW);
+	OP ("|=[1]", esil_mem_oreq1, 0, 2, OT_MATH | OT_MEMR | OT_MEMW);
+	OP ("|=[2]", esil_mem_oreq2, 0, 2, OT_MATH | OT_MEMR | OT_MEMW);
+	OP ("|=[4]", esil_mem_oreq4, 0, 2, OT_MATH | OT_MEMR | OT_MEMW);
+	OP ("|=[8]", esil_mem_oreq8, 0, 2, OT_MATH | OT_MEMR | OT_MEMW);
+	OP ("^=[]", esil_mem_xoreq, 0, 2, OT_MATH | OT_MEMR | OT_MEMW);
+	OP ("^=[1]", esil_mem_xoreq1, 0, 2, OT_MATH | OT_MEMR | OT_MEMW);
+	OP ("^=[2]", esil_mem_xoreq2, 0, 2, OT_MATH | OT_MEMR | OT_MEMW);
+	OP ("^=[4]", esil_mem_xoreq4, 0, 2, OT_MATH | OT_MEMR | OT_MEMW);
+	OP ("^=[8]", esil_mem_xoreq8, 0, 2, OT_MATH | OT_MEMR | OT_MEMW);
+	OP ("&=[]", esil_mem_andeq, 0, 2, OT_MATH | OT_MEMR | OT_MEMW);
+	OP ("&=[1]", esil_mem_andeq1, 0, 2, OT_MATH | OT_MEMR | OT_MEMW);
+	OP ("&=[2]", esil_mem_andeq2, 0, 2, OT_MATH | OT_MEMR | OT_MEMW);
+	OP ("&=[4]", esil_mem_andeq4, 0, 2, OT_MATH | OT_MEMR | OT_MEMW);
+	OP ("&=[8]", esil_mem_andeq8, 0, 2, OT_MATH | OT_MEMR | OT_MEMW);
+	OP ("+=[]", esil_mem_addeq, 0, 2, OT_MATH | OT_MEMR | OT_MEMW);
+	OP ("+=[1]", esil_mem_addeq1, 0, 2, OT_MATH | OT_MEMR | OT_MEMW);
+	OP ("+=[2]", esil_mem_addeq2, 0, 2, OT_MATH | OT_MEMR | OT_MEMW);
+	OP ("+=[4]", esil_mem_addeq4, 0, 2, OT_MATH | OT_MEMR | OT_MEMW);
+	OP ("+=[8]", esil_mem_addeq8, 0, 2, OT_MATH | OT_MEMR | OT_MEMW);
+	OP ("-=[]", esil_mem_subeq, 0, 2, OT_MATH | OT_MEMR | OT_MEMW);
+	OP ("-=[1]", esil_mem_subeq1, 0, 2, OT_MATH | OT_MEMR | OT_MEMW);
+	OP ("-=[2]", esil_mem_subeq2, 0, 2, OT_MATH | OT_MEMR | OT_MEMW);
+	OP ("-=[4]", esil_mem_subeq4, 0, 2, OT_MATH | OT_MEMR | OT_MEMW);
+	OP ("-=[8]", esil_mem_subeq8, 0, 2, OT_MATH | OT_MEMR | OT_MEMW);
```

Comment:
zillions of warnings
![IMAGE 2019-06-20 12:15:05](https://user-images.githubusercontent.com/917142/59841641-0a8c9780-9355-11e9-8872-d1bd7d9810e6.jpg)